core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -71,6 +85,7 @@ class UserServiceSpec extends FlatSpecLike with TestDriverComponent with Mockito
     withEmptyTestDatabase { dataSource: SlickDataSource =>
       val project = defaultBillingProject.copy(googleProjectNumber = None)
       runAndWait(rawlsBillingProjectQuery.create(project))
+      runAndWait(workspaceQuery.save(workspace))
```

Comment:
why add this line?

Reply:
We want to test when there is a workspace exists. This line will insert a workspace with the billing project name.

Reply:
Removed